### PR TITLE
4.7.1 - v4 Migration Cleanup

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR part of issue #83 

- [ ] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

- Fix linting issues
- Fix dead links

### CODE REVIEW

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md [hunk 1]
```diff
@@ -131,100 +133,96 @@ When analyzing the collected values, the tester should try to figure out all var
...
+### Videos
 
-\
-**Videos**\
-\* Session Hijacking in Webgoat Lesson - <http://yehg.net/lab/pr0js/training/view/owasp/webgoat/WebGoat_SessionMan_SessionHijackingWithJHijack/>
+- [Session Hijacking in Webgoat Lesson](http://yehg.net/lab/pr0js/training/view/owasp/webgoat/WebGoat_SessionMan_SessionHijackingWithJHijack/)
 
-Related Security Activities
----------------------------
+## Related Security Activities
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md]: This section requires some URL fixing as well

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md [hunk 2]
```diff
@@ -131,100 +133,96 @@ When analyzing the collected values, the tester should try to figure out all var
...
 
--   OWASP Zed Attack Proxy Project (ZAP) - <https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project> - features a session token analysis mechanism.
--   Burp Sequencer - <http://www.portswigger.net/suite/sequencer.html>
--   Foundstone CookieDigger - <http://www.mcafee.com/us/downloads/free-tools/cookiedigger.aspx>
--   YEHG's JHijack - <https://www.owasp.org/index.php/JHijack>
+### Whitepapers
 
-References
-----------
+- RFC 2965 “HTTP State Management Mechanism”
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md]: Can you update the references to have valid links and follow best practices for URL markdown?

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7_Testing_for_Session_Management.md [hunk 3]
```diff
@@ -1,10 +1,8 @@
...
+HTTP is a stateless protocol, meaning that web servers respond to client requests without linking them to each other. Even simple application logic requires a user's multiple requests to be associated with each other across a "session”. This necessitates third party solutions – through either Off-The-Shelf (OTS) middleware and web server solutions, or bespoke developer implementations. Most popular web application environments, such as ASP and PHP, provide developers with built-in session handling routines. Some kind of identification token will typically be issued, which will be referred to as a “Session ID” or Cookie.
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7_Testing_for_Session_Management.md]: I can't put a comment on line 9. Please remove the `\` and add a newline

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md [hunk 4]
```diff
@@ -131,100 +133,95 @@ When analyzing the collected values, the tester should try to figure out all var
...
+
+More information here: [Testing for cookies attributes](4.7.2_Testing_for_Cookies_Attributes_OTG-SESS-002.md)
 
-More information here: [Testing for cookies attributes](Testing_for_cookies_attributes_(OWASP-SM-002) "wikilink")
+## Tools
 
-Tools
------
+- [OWASP Zed Attack Proxy Project (ZAP)](https://www.owasp.org/index.php/OWASP_Zed_Attack_Proxy_Project) - features a session token analysis mechanism.
+- [Burp Sequencer](http://www.portswigger.net/suite/sequencer.html)
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md]: Needs updating

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md [hunk 5]
```diff
@@ -131,100 +133,95 @@ When analyzing the collected values, the tester should try to figure out all var
...
 
-See the OWASP articles on [Session Management Countermeasures](:Category:Session_Management "wikilink").
+See the OWASP articles on [Session Management Countermeasures](https://www.owasp.org/index.php/Category:Session_Management).
 
 ### How to Avoid Session Management Vulnerabilities
 
-See the [OWASP Development Guide](:Category:OWASP_Guide_Project "wikilink") article on how to [Avoid Session Management](Session_Management "wikilink") Vulnerabilities.
+See the [OWASP Development Guide](https://www.owasp.org/index.php/Category:OWASP_Guide_Project) article on how to Avoid Session Management Vulnerabilities.
 
 ### How to Review Code for Session Management| Vulnerabilities
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md]: Remove the `|` in the section title

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md (hunk 4 above)
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md]: @ThunderSon Updated with fix

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md (hunk 5 above)
Comment (rejahrehim) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema_OTG-SESS-001.md]: Done

**Pull request merged**